src/resources/views/crud/fields/fetch.blade.php

```diff
@@ -0,0 +1,291 @@
+@php
```

Comment:
@tabacitu what do we do with this ```fetch``` field type? Why would someone ever use this one instead of ```fetch_or_create```?

Should we make this independent from relationships? So people use it with custom APIs?

Reply:
Update: we decided to move it to the /relationship folder.